.docker/ci-shadow-fixed/Dockerfile

```diff
@@ -7,11 +7,9 @@ MAINTAINER Dave Coleman dave@picknik.ai
 # Switch to ros-shadow-fixed
 RUN echo "deb http://packages.ros.org/ros-shadow-fixed/ubuntu `lsb_release -cs` main" | tee /etc/apt/sources.list.d/ros-latest.list
 
-# Upgrade packages to ros-shadow-fixed and clean apt-cache within one command
+# Upgrade packages to ros-shadow-fixed and clean apt-cache within one RUN command
 RUN apt-get -qq update && \
     apt-get -qq dist-upgrade && \
+
```

Comment:
adding spaces causes warnings in Docker as it will no longer be supported in future versions of docker, please revert